config/rbac/service_account.yaml

```diff
@@ -3,7 +3,7 @@ kind: ServiceAccount
 metadata:
   labels:
     app.kubernetes.io/name: serviceaccount
-    app.kubernetes.io/instance: controller-manager
+    app.kubernetes.io/instance: controller-manager-sa
```

Comment:
Have you tested upgrade path with this? Will there be 2 SA after we upgrade ?

Reply:
When the Operator is upgraded it doesn't change the name of the SA as the name is fixed to `controller-manager`